includes/admin/ShipmentOptions.php

```diff
@@ -0,0 +1,332 @@
+                "name"       => "[shipment_options][return_shipment]",
+                "type"       => "toggle",
+                "label"      => __("Return if no answer", "woocommerce-myparcel"),
+                "value"      => $shipmentOptions['isReturn'],
+                "help_text"  => __(
+                    "By default, a parcel will be offered twice. After two unsuccessful delivery attempts, the parcel will be available at the nearest pickup point for two weeks. There it can be picked up by the recipient with the note that was left by the courier. If you want to receive the parcel back directly and NOT forward it to the pickup point, enable this option.",
+                    "woocommerce-myparcel"
+                ),
+                "conditions" => [
+                    $conditionPackageTypePackage,
+                    $conditionDeliveryTypeDelivery,
+                    $conditionCarrierDefault,
+                ],
+            ],
+            [
+                "name"       => "[shipment_options][insured]",
+                "type"       => "toggle",
+                "label"      => __("Insured", "woocommerce-myparcel"),
+                "value"      => (bool) $shipmentOptions['insurance'],
+                "conditions" => [
+                    $conditionPackageTypePackage,
+                    $conditionDeliveryTypeDelivery,
+                    [
+                        "parent_name"  => "[carrier]",
+                        "type"         => "disable",
+                        "parent_value" => WCMP_Data::DEFAULT_CARRIER,
+                        "set_value"    => WCMP_Settings_Data::DISABLED,
+                    ],
+                ],
+            ],
+            [
+                "name"       => "[shipment_options][insured_amount]",
+                "type"       => "select",
+                "label"      => __("Insurance amount", "woocommerce-myparcel"),
+                "options"    => WCMP_Data::getInsuranceAmount(),
+                "value"      => (int) $shipmentOptions['insurance'],
+                "conditions" => [
+                    $conditionPackageTypePackage,
+                    $conditionDeliveryTypeDelivery,
+                    "[shipment_options][insured]",
+                ],
+            ],
+            [
+                "name"  => "[shipment_options][label_description]",
+                "type"  => "text",
+                "label" => __("Custom ID (top left on label)", "woocommerce-myparcel"),
+                "value" => $shipmentOptions['label_description'] ?? null,
+            ],
+        ];
+    }
+
+    /**
+     * Filters out rows that should not be shown if the shipment is sent to the home country.
+     *
+     * @param string $cc
+     * @param array  $rows
+     *
+     * @return array
+     */
+    public static function filterRowsByCountry(string $cc, array $rows): array
+    {
+        $isDefaultCountry = WCMP_Data::DEFAULT_COUNTRY_CODE === $cc;
+
+        if ($isDefaultCountry) {
```

Comment:
```suggestion
        if (WCMP_Data::DEFAULT_COUNTRY_CODE === $cc) {
```
Wat op regel 267 staat kan dan weg.